src/modules/core/components/Tag/Tag.css

```diff
@@ -46,6 +46,12 @@
   composes: baseStyles;
 }
 
+.themeBanned {
+  composes: baseStyles;
+  background-color: var(--banned);
+  color: var(--danger);
```

Comment:
It should be --pink (by the look from the specs)

Reply:
Done!